.github/workflows/npm-publish.yaml

```diff
@@ -0,0 +1,31 @@
+name: npm-publish
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version you intend to publish (eg x.y.z)
+  release:
+    types: [ created ]
+
+jobs:
+  publish:
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup-node
+        uses: actions/setup-node@v3
```

Comment:
Woops, this action wasn't supposed to be committed since it clearly doesn't do what i want. Will update this one.